great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+	  <header>
+ 	<section class="section1">
+	  <nav>
+	 Services
+  	Product
+  	Vision
+	Features
+  	About
+  	Contact
+	</nav>
+  
 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </section>
+  
+	  <section class="section2">
+	  <h1 class="main-header"> 
+ 
+		  <div class="innovation">Innovation</div>    
+		<div class="on">On</div>
+		
+	  	   <div class="Demand">Demand</div>
+		  
+		  <button>
+		Get Started  
+		  
+	</button> 
+		  
+		  </h1>
+
+ 	
+
+  <img src="img/header-img.png" alt="Image-of-a-code-snippet">
```

Comment:
If you visually look at the design file, you have the `h1` on the top left, the `button` on the bottom left. Both of them are centered vertically in the section. The image is on the right, also centered vertically in the section.

This problem is kind of hard to solve if you try to target the three elements individually. But what about if you put the `h1` and the `button` in its own `div`? Would that make it easier? :thinking: